db/migrate/20230118130656_remove_role_appointment_for_speech.rb

```diff
@@ -0,0 +1,14 @@
+# Zendesk ticket https://govuk.zendesk.com/agent/tickets/5182511
+# Need to unlink the speech from the role appointment.
+# This speech in published state is already linked to person's new role.
+# Because the speech is superseded and would raise the validation
+# "cannot be modified when edition is in the superseded state"
+# it must be saved without running the validations.
+
+class RemoveRoleAppointmentForSpeech < ActiveRecord::Migration[7.0]
+  def change
+    speech = Speech.find(1_216_085)
+    speech.role_appointment_id = nil
+    speech.save!(validate: false)
+  end
+end
```

Comment:
This causes issues locally where the data may not be present.

My recommendation would be to change to this:
```
class RemoveRoleAppointmentForSpeech < ActiveRecord::Migration[7.0]
  def change
    speech = Speech.find(1_216_085)
    if speech
      speech.role_appointment_id = nil
      speech.save!(validate: false)
    end
  end
end
```

Ideally, this could have been done better by the cli rather than a migration.